### ISSUE TITLE
Broken by update 100?

### ISSUE THREAD

**Issue by qur** [polarathene/biglobby#9]
Has this mod been broken by update 100?  It was working fine before the update, but now I can't seem to get it working ... it acts almost like the pdmod is not loaded.

**Comment by polarathene** [comment]
I would assume so, the update changed many asset bundles, the pdmod affects one of those so if that happened it'd be broken. The source is available to recreate your own pdmod, else I'll do it when I can find the time(may be up to a week).

**Comment by polarathene** [comment]
I've pushed the dev branch onto master with some new updates including the pdmod fix. I've not updated the ReadMe, v2 isn't properly done(checklist [here](https://github.com/polarathene/biglobby/issues/5)). It should be useable now at least, let me know if you run into any issues.

There is now a settings option for the amount of players allowed in the lobby you host. It defaults to 8, exceeding the total characters provided by the game may cause breakage until v2 is finished.

I created a release [here](https://github.com/polarathene/biglobby/releases/tag/v2.0.0-alpha).
